ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift

```diff
@@ -119,7 +119,11 @@ final class SidebarViewController: UITableViewController {
             let vc = SponsorViewController()
             transition(to: vc)
         case .contributor:
-            break
+            if rootViewController.viewControllers.first is ContributorViewController {
+                break
+            }
+            let contributorViewController = ContributorViewController.instantiate()
+            rootViewController.pushViewController(contributorViewController, animated: true)
```

Comment:
In my opinion, the view moved from sidebar should have `menuButton` and not to move with navigation.
And views moved from sidebar's transition way should be same I think.

So, can you move view with `transition(to:)` function? 🙏

Reply:
Sorry, I didn't see other branches. I'll fix like that.